test/testdrive/fetch-tail-timestamp-zero.td

```diff
@@ -27,6 +27,7 @@
 
 > FETCH 2 FROM c1 WITH (timeout = '60s')
 18446744073709551615 false 1 123
+18446744073709551615 true <null> <null>
```

Comment:
Well, this is confusing. The line with `true` should come first, right? And it probably does, but testdrive is not order sensitive. In this case, I think we _do_ want to assert on the order, so perhaps best to split this into two separate `FETCH 1 FROM` commands that perform the assertions in the right order?

Reply:
yeah, good point